### DISCUSSION TITLE
Show "Prepare" tab instead of "Home" tab at startup

### DISCUSSION THREAD

**Discussion by VagnPetersen** [OrcaSlicer/OrcaSlicer#2377] [Q&A]
Running Orca slicer 1.7.0 on Windows 11 and I love it!
However, I am a little annoyed that the default view at startup is the "home" tab.
Is there a way to have the program starting up displaying the "Prepare" tab instead?

**Comment by LJKraus** [comment]
Might be best to also request this option in the Studio base code, and let it become part of Orca in a future release.

https://github.com/bambulab/BambuStudio/issues

**Comment by LJKraus** [comment]
This is topic is showing as recent activity, but I see none?

In any case, this became an OrcaSlicer option several releases ago:

![image](https://github.com/user-attachments/assets/b4f13a7e-f643-44ae-83de-786c48010af3)
